pom.xml

```diff
@@ -356,7 +356,7 @@
             <dependency>
                 <groupId>org.checkerframework</groupId>
                 <artifactId>checker-qual</artifactId>
-                <version>3.32.0</version>
+                <version>3.33.0</version>
```

Comment:
How did this change end up in this PR? :thinking:

Reply:
Nice comment.
A wild commit appeared from a forcedpush I assume.
See: [64f9d6b](https://github.com/PicnicSupermarket/error-prone-support/pull/563/commits/64f9d6b7a23876acb5e7de23824d89d100394bd4).

Reply:
Maybe stupid question.
@Stephan202 can I drop the commit ^ from this PR ? 🤔

Reply:
Yes you can :).

Reply:
I can't do that apparently 🤔 😓

Reply:
![Screenshot 2023-04-05 at 13 46 21](https://user-images.githubusercontent.com/110535847/230071061-93ef2ca8-460d-4916-8ce8-2698eca00a00.png)
Not from Intellij atleast 🤔

Reply:
Weird, if I diff locally, I don't get these changes 🤔.

Reply:
I think the state of the PR is fine but that GitHub doesn't have the correct state registered 🤔.

Reply:
But I do see them locally 😬 
Did you pull changes ?

Reply:
@mohamedsamehsalah perhaps your branch is outdated? I don't see it locally.

Reply:
```
➜  error-prone-support git:(mohamedsamehsalah/mono-just-or-empty-from-optional) git fetch && git pull
Already up to date.
```

it could be I am doing something wrong but I cant pin point what it is 🤷

Reply:
This  weird behavior is also flagged in our internal Slack. It is suggested to close and reopen.. So will try that.

Reply:
Seems fixed now.

Reply:
Thx for figuring it out. :+1: